addons/mail/static/src/models/thread_view/thread_view.js

```diff
@@ -217,7 +227,12 @@ function factory(dependencies) {
          * Most recent message in this ThreadView that has been shown to the
          * current partner.
          */
```

Comment:
PR looks much nicer without the rename, but as discussed, add comment about scope of field into the field docstring too.